lib/spack/spack/test/cmd/load.py

```diff
@@ -136,3 +137,66 @@ def test_unload_fails_no_shell(install_mockery, mock_fetch, mock_archive,
 
     out = unload('mpileaks', fail_on_error=False)
     assert "To initialize spack's shell commands" in out
+
+
+def test_load_installed_package_not_in_repo(install_mockery,
+                                            mock_fetch, monkeypatch):
+    """Test that spack load can still load a package that is no
+       longer in the builtin repo, but has an installation."""
+    # Get a basic concrete spec for the trivial install package.
+    spec = Spec('trivial-install-test-package')
+    spec.concretize()
+    assert spec.concrete
+
+    # Get the package
+    pkg = spec.package
+
+    def find_nothing(spec, *args):
+        monkeypatch.delattr(spack.repo.path, 'get')
+        pkg = spack.repo.path.get(spec)
```

Comment:
This assignment can be inlined in the return statement, since `pkg` isn't used anywhere inside this method.